Bindings/Python/simple-arm.py

```diff
@@ -0,0 +1,132 @@
+# You may obtain a copy of the License at                                 %
+# http://www.apache.org/licenses/LICENSE-2.0.                             %
+#                                                                         % 
+# Unless required by applicable law or agreed to in writing, software     %
+# distributed under the License is distributed on an "AS IS" BASIS,       %
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or         %
+# implied. See the License for the specific language governing            %
+# permissions and limitations under the License.                          %
+# ----------------------------------------------------------------------- %
+
+# simple-arm.py                                                        
+# Author: Neil Dhir
+
+import opensim as osim
+
+# Define global model where the arm lives.
+arm = osim.Model()
+
+# ---------------------------------------------------------------------------
+# Create two links, each with a mass of 1 kg, centre of mass at the body's
+# origin, and moments and products of inertia of zero.
+# ---------------------------------------------------------------------------
+
+# chrisdembia: use 4 spaces instead of tabs
+humerus = osim.Body('humerus',
+	1.0,
+	osim.Vec3(0, 0, 0),
+	osim.Inertia(0, 0, 0))
+radius = osim.Body('radius',
+	1.0,
+	osim.Vec3(0, 0, 0),
+	osim.Inertia(0, 0, 0))
+
+# ---------------------------------------------------------------------------
+# Connect the bodies with pin joints. Assume each body is 1m long.
+# ---------------------------------------------------------------------------
+
+shoulder = osim.PinJoint("shoulder",
+	arm.getGround(), 	    		# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 0, 0),
+	humerus, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 1, 0))
+
+elbow = osim.PinJoint("elbow",
+	humerus, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 0, 0),
+	radius, 				# PhysicalFrame
+	osim.Vec3(0, 0, 0),
+	osim.Vec3(0, 1, 0))
+
+# ---------------------------------------------------------------------------
+# Add a muscle that flexes the elbow (actuator for robotics people).
+# ---------------------------------------------------------------------------
+
+biceps = osim.Millard2012AccelerationMuscle("biceps", 	# 	Muscle name
+   	200.0, 	 	#	Max isometric force
+    	0.6, 		# 	Optimal fibre length
+    	0.55, 		# 	Tendon slack length
+    	0.0)		# 	Pennation angle
+biceps.addNewPathPoint("origin",						
+   	humerus, 						
+   	osim.Vec3(0, 0.8, 0))			
+
+biceps.addNewPathPoint("insertion",
+   	radius,
+   	osim.Vec3(0, 0.7, 0))
+
+# # ---------------------------------------------------------------------------
+# # Add a controller that specifies the excitation of the muscle.
+# # ---------------------------------------------------------------------------
+
+brain = osim.PrescribedController()
+brain.addActuator(biceps)
+brain.prescribeControlForActuator(1,					
```

Comment:
Wait, no? Neither using `0` nor `biceps` actually work. Both yield the same error message:

`[Finished in 0.8s with exit code -11]` 

still don't know what that means. Literally, the full thing it writes out is simply:

```
[Finished in 0.7s with exit code -11]
[shell_cmd: python -u "/Users/wagglefoot/cloud/control/src/muscles/python/opensim/simple-arm.py"]
[dir: /Users/wagglefoot/cloud/control/src/muscles/python/opensim]
[path: /Library/Frameworks/Python.framework/Versions/2.7/bin:/opt/local/bin:/opt/local/sbin:/Users/neildhir/.rvm/gems/ruby-2.2.1/bin:/Users/wagglefoot/.rvm/gems/ruby-2.2.1@global/bin:/Users/wagglefoot/.rvm/rubies/ruby-2.2.1/bin:/opt/local/bin:/opt/local/sbin:/Library/Frameworks/Python.framework/Versions/3.3/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/git/bin:/Library/TeX/texbin:/Users/wagglefoot/.rvm/bin:/Users/wagglefoot/.rvm/bin] 
```

and issues two empty files `out.log` and `err.log`. That is the complete thing.